docs/source/roadmaps/may-2024.md

```diff
@@ -0,0 +1,117 @@
+data. For more details on this specific project, see the [movement roadmap](https://movement.neuroinformatics.dev/community/roadmap.html).
+
+**Q3 2025**
+
+- Support all common post pose-estimation analysis tasks
+- Support for single experiments and complex multi-animal, multi-session datasets 
+(by reading and writing to [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/) format)
+
+**No current ETA**
+
+- A common interface to machine-learning based video behavioural analysis tools
+- A common interface to machine-learning based behavioural classification tools
+
+## Extracellular electrophysiology analysis
+**Current status**
+
+We are currently building [spikewrap](https://github.com/neuroinformatics-unit/spikewrap) (built on top of 
+[SpikeInterface](https://spikeinterface.readthedocs.io/en/latest/)) for simple, routine analysis of extracellular 
+electrophysiology analysis. For more details on this specific project, see the
+[spikewrap roadmap]().
+
+**Q2 2025**
+
+- Easy to use tool for all common extracellular electrophysiology analysis steps
+- Support for single experiments and complex multi-animal, multi-session datasets
+  (by reading and writing to [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/) format)
+
+## Computational anatomy
+**Current status**
+
+Our computational anatomy work is based around the [BrainGlobe Initiative](https://brainglobe.info). BrainGlobe is
+centered around the [BrainGlobe Atlas API](https://brainglobe.info/documentation/brainglobe-atlasapi/index.html) which
+provides a common interface to multiple anatomical atlases. Currently, there are tools for:
+- 3D whole brain microscopy registration ([brainreg](https://brainglobe.info/documentation/brainreg/index.html))
+- 3D cell detection in large images ([cellfinder](https://brainglobe.info/documentation/cellfinder/index.html))
+- Analysis of arbitrary shaped structures in atlas space such as lesions and implanted devices
+  ([brainglobe-segmentation](https://brainglobe.info/documentation/brainglobe-segmentation/index.html))
+- Visualisation of atlas registered data ([brainrender](https://brainglobe.info/documentation/brainrender/index.html) &
+  [brainglobe-heatmap](https://brainglobe.info/documentation/brainglobe-heatmap/index.html))
+- Various small tasks such as [reorienting anatomical images](https://brainglobe.info/documentation/brainglobe-space/index.html)
+
+**Future**
+
+BrainGlobe is a large project with many short- and long-term aims. Our two main goals are to:
+- Allow for all neuroscience histology analysis and visualisation tasks to be carried out easily in a single
+  environment. This includes 2D & 3D data in all common animal models
+- Create an end-to-end pipeline to analyse mouse brains imaged with a [mesoSPIM](https://mesospim.org/)
+
+For full details, please see the [BrainGlobe roadmap](https://brainglobe.info/community/roadmaps/index.html).
+
+## Multimodal integration
```

Comment:
I thought about that, but is it slightly weird to talk about linking together tools that the reader may not have heard of yet?

Reply:
Hmm that's true, could it be more general (e.g. we want to do multimodal behaviour, ephys, integrate anatomy) and then introduce the tools we are building to achieve such goals (e.g. movement, spikewrap, brainglobe). And then below have summary roadmaps of the introduced tools? My guess is that people on this page would be mostly interested in the overarching NIU priorities, and might take a look at the tool-specific ones if of particular interest / relevance.

Reply:
in my view, a roadmap isn't much use unless it's (relatively) specific and has some concept of an ETA. Therefore, things like "integrate anatomy" are a bit vague, and don't tell the user what we're actually going to do. If they use another anatomy tool than BrainGlobe, they won't care if we combine BrainGlobe and spikewrap.

I think the answer may be to do add some kind of guiding principles somewhere else?

Reply:
> I think the high-level priorities of the NIU are very inspiring and researchers would be keen to hear more about them (i.e.. complete automation of multimodal analysis and integration with anatomy).

I saw this after my comment. I agree, but this should be elsewhere. That's not (in my view) a roadmap.

Reply:
Yes that's true, how about something that is like an overview but links to the sections below to add concreteness. I agree the roadmap needs to be specific and include ETA as a priority. But if I am imagining myself as a researcher who has no idea what we do and I visit the roadmap page, I don't really get a sense of what the team is about. It seems like we are doing a lot of unconnected things and then at the end, realise there is some deeper goal.

I don't think it would need to be very long, just a few sentences at the top to explain the vision. For example 

"Our ultimate goal at the NIU is to provide fully integrated, automated analysis for systems neuroscience data. As a researcher, this would mean inputting only a set of configurations before running the analysis, with results and qualtiy metrics output for checking. This will avoid individual researchers having to write custom preprocessing and analysis scripts that usually duplicate existing efforts, and allows researchers to work together on community tools. 

This is a long-term goal and we are working across multiple themes towards acheiving this. We are ensuring [data is organised in a standard way](), a critical step towards automation. Currently, we are focused on building pipelines for [behaviour](), [electrophysiology]() and [anatomy]() that can ingest data from disparate acquisition systems into a common interface. These are critical steps towards the longer-term goal of [multimodal integration](). The detailed roadmap is split into these themes below."

Reply:
Nice. I've added a version of that.